### DISCUSSION TITLE
每次邮箱发送2步验证码的时候,都会提示如图,但是实际是可以收到有效的验证码的

### DISCUSSION THREAD

**Discussion by g2112000290** [dani-garcia/vaultwarden#5074] [General]
每次邮箱发送2步验证的时候,都会提示如图,但是实际是可以收到有效的验证码的,请问是哪里出来了问题.总是弹出错误提示.
![搜狗截图20241012092219](https://github.com/user-attachments/assets/91c385af-e7fd-46e5-a5a7-114e25a8d20b)

**Comment by BlackDex** [comment]
See #4996

**Reply by g2112000290** [reply]
看完了所有 看不懂怎么解决,你能直接告诉我 改哪些东西吗,谢

**Reply by BlackDex** [reply]
There is no solution as if yet other then using a different smtp provider.
